### DISCUSSION TITLE
Shadows are not applied on Android

### DISCUSSION THREAD

**Discussion by keysoon** [tailwindlabs/tailwindcss#3879] [Help]
### What version of Tailwind CSS are you using?

2.0.3

### What version of Node.js are you using?

v12.16.1

### What browser are you using?

Chrome 

### What operating system are you using?

Android

### Reproduction repository

https://gitee.com/weiqs/ec-union

Shadows are rendered on iOS
![image](https://user-images.githubusercontent.com/7811765/112247888-ff843b00-8c8f-11eb-8053-cc4116ea1210.png)

Shadows do not work on Android
![image](https://user-images.githubusercontent.com/7811765/112248036-3b1f0500-8c90-11eb-84ad-936301884c9b.png)

**Comment by adamwathan** [comment]
Hey! There must be more going on here because shadows definitely work in Android:

https://play.tailwindcss.com/p5X9s8kfce

![telegram-cloud-photo-size-4-5987900296229140183-y](https://user-images.githubusercontent.com/4323180/112995247-07812680-9139-11eb-91ee-3cec9d7c8255.jpg)

The reproduction repo is private but for this I'd rather get a minimal reproduction via Tailwind Play. I suspect this is just an issue with how the page is structured and how things behave cross browser, which is ultimately not going to be something we could fix within Tailwind.

Going to transfer to our discussions since I don't really believe it's possible for this to be a bug with how our CSS is generated.
